src/utils/fetch.ts

```diff
@@ -22,6 +27,13 @@ const RATE_LIMIT_HEADER = 'X-Discogs-Ratelimit'
 /** Header set by Discogs API to indicate the number of remaining requests you are able to make in the existing rate limit window. */
 const RATE_LIMIT_REMAINING_HEADER = 'X-Discogs-Ratelimit-Remaining'
 
+let buffer: undefined | typeof Buffer
+try {
+  buffer = Buffer
+} catch {
+  buffer = undefined
+}
```

Comment:
What is the purpose of this?

Reply:
I'm guessing there was a situation in some browser where _accessing_ `Buffer` to calculate content length through. I honestly don't remember the details. Getting Chrome to be happy with those headers and their capitalization was a PITA. I myself would reject this without an explanatory comment about the circumstances of the error and the expected alternate behavior...